ksql-engine/src/test/java/io/confluent/ksql/ddl/commands/CreateTableCommandTest.java

```diff
@@ -137,6 +142,21 @@ public void shouldThrowOnOldWindowProperty() {
     createCmd();
   }
 
+  @Test
+  public void testCreateAlreadyRegisteredTableThrowsException() {
```

Comment:
Likewise:

```java
    // Given:
    givenProperties(propsWith(ImmutableMap.of()));
    final CreateTableCommand cmd = createCmd();
    cmd.run(metaStore);

    // Then:
    expectedException.expectMessage(
        "Failed to create because a TABLE with name 'name' already exists.");

     // When:
    cmd.run(metaStore);
```